src/main/java/com/julienviet/pgclient/impl/codec/DataType.java

```diff
@@ -435,10 +435,20 @@ public void encodeText(Buffer value, ByteBuf buff) {
   public static DataType<Object> MACADDR8 = new DataType<>(Object.class,774);
 
   // UUID
-  public static DataType<String> UUID = new DataType<String>(String.class, 2950) {
+  public static DataType<java.util.UUID> UUID = new DataType<java.util.UUID>(java.util.UUID.class, 2950) {
     @Override
-    public String decodeText(int len, ByteBuf buff) {
-      return buff.readCharSequence(len, StandardCharsets.UTF_8).toString();
+    public java.util.UUID decodeText(int len, ByteBuf buff) {
+      return java.util.UUID.fromString(buff.readCharSequence(len, StandardCharsets.UTF_8).toString());
```

Comment:
shouldn't we use instead byte[] ?

Reply:
Here we are getting data as text

Reply:
ah ok makes sense :-) and the other is using 2 longs which makes 16 bytes

Reply:
and anyway it's not really going to be a bottleneck

Reply:
Yes :)